src/constants.js

```diff
@@ -1,4 +1,4 @@
 export const TYPING_INDICATOR_TTL = 1500
 export const TYPING_INDICATOR_LEEWAY = 500
 export const SET_CURSOR_WAIT = 500
-export const SUBSCRIPTION_TIMEOUT = 5000
+export const CONNECTION_TIMEOUT = 10 * 1000
```

Comment:
Perhaps we should call this `DEFAULT_CONNECTION_TIMEOUT`?

Reply:
good idea. Makes it clear that it can be changed, unlike the others.